.travis.yml

```diff
@@ -187,10 +187,18 @@ matrix:
       env: TEST_SUITE=plugins DB=postgres PERCY_TOKEN=${PLUGIN_PERCY_TOKEN}
 
     - python: 2.7
-      name: 'Frontend'
+      name: 'Frontend [test]'
       env: TEST_SUITE=js
       before_install:
-        - *base_install
```

Comment:
Does our frontend test suite actually need this? It has no python/pip deps. /cc @joshuarli

Reply:
Doesn't look like it needs it to me. Should work.